tests/subsys/llext/simple/boards/qemu_arc_qemu_arc_em.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_FILE_SYSTEM=y
+CONFIG_FILE_SYSTEM_LITTLEFS=y
+CONFIG_FS_LITTLEFS_FMP_DEV=y
+CONFIG_FLASH_MAP=y
+CONFIG_FLASH=y
+CONFIG_FLASH_SIMULATOR=y
+# remove when MPU is enabled
+CONFIG_ARC_MPU_ENABLE=n
```

Comment:
Sorry I hadn't spotted it earlier, but this will blanket disable MPU for the `qemu_arc_em` board. 
The same thing is accomplished (when appropriate) by the line in `no_mem_protection.conf`, so this should be removed.